app/lib/features/news/widgets/news_side_bar.dart

```diff
@@ -47,13 +49,40 @@ class NewsSideBar extends ConsumerWidget {
     );
 
     return Column(
+      mainAxisAlignment: MainAxisAlignment.end,
       children: <Widget>[
-        const Spacer(),
-        LikeButton(
-          likeCount: news.likesCount().toString(),
-          style: style,
-          color: fgColor,
-          index: index,
+        SizedBox(
+          height: 75,
+          width: 75,
+          child: LikeButton(
+            isLiked: isLikedByMe.valueOrNull ?? false,
+            likeCount: reactions.valueOrNull != null
+                ? reactions.valueOrNull!.length.toString()
+                : '0',
+            style: style,
+            color: fgColor,
+            index: index,
+            onTap: () async {
+              final client = ref.read(alwaysClientProvider);
+              final status = await news.myLikeStatus();
+              final manager = await news.reactions();
+              debugPrint('STATUS: $status');
+              if (!status) {
+                final eventId = await manager.sendReaction(
+                  news.eventId().toString(),
+                  '\u{dd25}',
+                );
+
+                await client.waitForReaction(eventId.toString(), null);
+              } else {
+                final reactions = await manager
+                    .reactionEntries()
+                    .then((value) => value.toList());
+                final eventId = reactions[0].eventIdStr();
+                await manager.redactReaction(eventId, '', '');
+              }
```

Comment:
the manager is specific to this news item ... why does the API user have to figure out if there was `like` by me and additionally look up the specific entry and redact that... This is all code that should be _in_ the rust SDK API not here...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/38a5cbc2d7976cf2464c65823f99898495a46390